src/FlipMove.js

```diff
@@ -69,7 +69,7 @@ class FlipMove extends Component {
     // can complete. Because we cannot mutate props, we make `state` the source
     // of truth.
     this.state = {
-      children: props.children.map(child => ({
+      children: Children.toArray(props.children).map(child => ({
```

Comment:
The problem is that `propConverter` composes around the `FlipMove` component.  That means the children object passes back through the React virtual DOM and so the children prop passed to `FlipMove` may not be the same one that came from `propConverter`.  Unfortunately, we have to do all of the children conversions in `FlipMove` instead of `propConverter` because `children` is a special prop.